apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -719,6 +720,24 @@ export default function teacherSections(state = initialState, action) {
       }
     }
 
+    let assignmentData = {
+      section_id: section.id,
+      section_creation_timestamp: section.createdAt
+    };
+    if (section.scriptId !== state.initialScriptId) {
+      assignmentData.script_id = section.scriptId;
+    }
+    if (section.courseId !== state.initialCourseId) {
+      assignmentData.course_id = section.courseId;
+    }
+    if (assignmentData.script_id || assignmentData.course_id) {
+      firehoseClient.putRecord({
+        study: 'assignment',
```

Comment:
let's also add in a study group so we can capture before our change (even though we won't be in this state very long) and after our change. Something like: `study_group: 'v0',`

Reply:
are we still pre-change? Not sure if this suggestion is still relevant.

Reply:
Still pre-change, but hopefully not for very long. We can update this again to v1 when the experiment flag is removed.